math/minuit2/StandAlone.cmake

```diff
@@ -12,8 +12,8 @@ endif()
 include(copy_standalone.cmake)
 
 # If MAKE_STANDALONE, copy these files in
-copy_standalone(../../../build . version_number)
-copy_standalone(../../.. . LICENSE LGPL2_1.txt)
+copy_standalone(../../build . version_number)
+copy_standalone(../.. . LICENSE LGPL2_1.txt)
```

Comment:
What's wrong with `${CMAKE_COMMAND} -E copy [src] [dst]` or even `file(COPY ...)`?

Reply:
This one makes sure the target directory exists, adds it to the purge target, and most importantly does not make the copy unless this is inside ROOT. This is what allows this to work when it is actually "standalone".

Reply:
Internally it uses `configure_file(... COPYONLY)`, but it could use either of those two as well.